cmd/grpcurl/grpcurl.go

```diff
@@ -313,6 +320,22 @@ func main() {
 		return cc
 	}
 
+	if *expandHeaders {
+		var err error
+		addlHeaders, err = grpcurl.ExpandHeaders(addlHeaders)
```

Comment:
Let me know if you wanted this DRY'd, was on the fence.

Reply:
This seems fine.